addons/point_of_sale/static/src/js/Chrome.js

```diff
@@ -51,88 +59,13 @@ export class Chrome extends Component {
             }
         });
 
-        // This is not done in onWillStart because we want to show the loader immediately
-        this.start();
-    }
-    async start() {
-        await this.pos.globalState.load_server_data();
-        if (this.pos.globalState.config.use_proxy) {
-            await this.pos.connectToProxy();
-        }
-        this._closeOtherTabs();
-        this.pos.uiState = "READY";
-        const { name, props } = this.startScreen;
-        this.pos.showScreen(name, props);
-        this.runBackgroundTasks();
+        onMounted(() => {
+            this.props.loader.isShown = false;
```

Comment:
I remember that doing something like this is discouraged. And instead doing this direct mutation to the props, the component should accept a method that it should call. But to be honest, I'm not against this diff.